k8s/dev/migration.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: batch/v1
```

Comment:
note: this will make it potentially wierd to run kubectl apply -f dev/. Maybe we can move to /migrate